packages/types-dev/objects.d.ts

```diff
@@ -677,7 +675,8 @@ declare global {
             unsafePerm?: true;
             /** The available version in the ioBroker repo. */
             version: string;
-            visWidgets?: Record<string, VisWidget>;
+            /** The vis widget does not support the listed major versions of vis */
+            visWidgets?: Record<string, VisWidget> | { ignoreInVersions?: number[] };
```

Comment:
Now you can either define via widgets or ignore I do not think it is the intention. See also my previous comment.

Reply:
Okay, please confirm: 

Valid: 
```json
....
"visWidgets": {
    "ignoreInVersions": [2]
}
.... 
```

Valid: 
....
"visWidgets": {
    "mySuperWidget": { ... }
}
.... 
```

Invalid:
....
"visWidgets": {
    "mySuperWidget": { ... },
    "ignoreInVersions": [2]
},
.... 
```

Maybe add some type tests to `types-public` package in `index.test-d.ts` to ensure it works like intended.